resolwe_bio/processes/expression/featureCounts.yml

```diff
@@ -42,14 +42,40 @@
             Indicate if strand-specific read counting should be performed. For
             paired-end reads, strand of the first read is taken as the strand
             of the whole fragment. FLAG field is used to tell if a read is
-            first or second read in a pair.
+            first or second read in a pair. Automated strand detection is enabled
+            using the [Salmon](https://salmon.readthedocs.io/en/latest/library_type.html)
+            tool's build-in functionality. To use this option, cDNA (transcriptome)
+            index file crated using the Salmon indexing tool must be provided.
           choices:
```

Comment:
Consider changing the `default` attribute to `auto` here as well. The same applies also to the workflow and other associated processes.

Reply:
I have deliberately left the basic process behaviour to be as similar as is expected from the featureCounts tool. Auto is than used in workflows and elsewhere where it makes sense.